src/shared/util/spawn.js

```diff
@@ -13,14 +13,18 @@ specific language governing permissions and limitations under the License.
 import cp from 'child_process';
 
 import colors from 'colour';
+import { IS_PACKAGED_BUILD } from '../build-info';
 
 export const spawn = (command, main, args, { logger }, options = {}) => new Promise((resolve, reject) => {
   logger.log('Spawning',
     colors.cyan(command),
     colors.blue(main),
     colors.yellow(args.join(' ')));
 
-  const child = cp.spawn(command, [main, ...args], { stdio: 'inherit', ...options });
+  const stdio = process.platform === 'win32' && IS_PACKAGED_BUILD
+    ? 'ignore'
+    : 'inherit';
```

Comment:
How about just passing this as part of `options` directly on invocation, instead of hardcoding it in this (presumably generic) `spawn` function? Add a comment about it too.

Or, can you detail why it would be better to have this here? AFAICT it's a purely electron thing.